packages/docs/README.md

```
@@ -0,0 +1,191 @@
+The VersionPicker component is embedded in each package's API Reference page (`docs/*/api.mdx`).
+
+It will only work in builds that generate `versions.json`:
+- ✅ `pnpm build` - Full versioned docs, includes versions.json
+- ✅ CI builds - Uses `build`
+- ❌ `pnpm build:prose` - Prose only, no API docs, no versions.json
+- ❌ `pnpm start:full` - Dev mode with current API, no versions.json
+
+## Build Scripts
+
+| Script | Purpose |
+|--------|---------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)